src/redux/sagas/base.js

```diff
@@ -1,17 +1,22 @@
 import {
   put, takeEvery, takeLatest,
 } from 'redux-saga/effects';
-import { blockchainActions } from '../actions';
+import { blockchainActions, routeActions } from '../actions';
 
 function errorHandler(onFailure, worker) {
   function* errorHandledWorker(action) {
     try {
       yield* worker(action);
     } catch (e) {
       // eslint-disable-next-line no-console
-      console.error(e);
+      const isWallet = !e.stack.includes('Error: web3FromAddress: Unable to find injected');
```

Comment:
This is very defensive and should be handled by reviewing the codebase and ensuring this doesn't happen to users. At the very least, this should be an "instanceof" check for error type if possible.

Reply:
For instance, what if we upgrade polkadot library and  they change the error messages? This would stop working then.

Reply:
If this is about accessing routes without an appropriate browser/wallet combination, I think react-router-dom library already has a way of validating a route, maybe try that?